src/utils/eventDispatcher.test.ts

```diff
@@ -0,0 +1,61 @@
+import assert from 'assert';
+import EventDispatcher from './eventDispatcher';
+
+test('should emit event', () => {
```

Comment:
Isn't it more natural to use `it` and `describe`?

```suggestion
describe('EventDispatcher', () => {
  it('should emit event', () => {
```

https://create-react-app.dev/docs/running-tests/#writing-tests

Reply:
Thank you for your comment :)
I agree with your words!